### PR TITLE
update etcd makefile to build v3.5.0 image

### PR DESCRIPTION
**Pull request by pacoxu**
/kind feature
Ref #102294
/cc serathius BenTheElder
/area release-eng
/priority important-soon

```release-note
NONE
```

### CODE REVIEW

File: cluster/images/etcd/Makefile [hunk 1]
```diff
@@ -26,10 +26,10 @@
...
 # need etcd and etcdctl binaries for backward compatibility reasons.
 # That binary will be set to the last version from $(BUNDLED_ETCD_VERSIONS).
-BUNDLED_ETCD_VERSIONS?=3.0.17 3.1.12 3.2.24 3.3.17 3.4.13 3.5.0-rc.0
+BUNDLED_ETCD_VERSIONS?=3.0.17 3.1.12 3.2.24 3.3.17 3.4.13 3.5.0
 
 # LATEST_ETCD_VERSION identifies the most recent etcd version available.
-LATEST_ETCD_VERSION?=3.5.0-rc.0
+LATEST_ETCD_VERSION?=3.5.0
 
 # REVISION provides a version number for this image and all it's bundled
```
Comment (serathius) [inline cluster/images/etcd/Makefile:34]: Please reset the revision

**Review by BenTheElder — APPROVED** [review]
needs the revision reset but otherwise
/approve
LGTM pending that

File: cluster/images/etcd/Makefile (hunk 1 above)
Comment (pacoxu) [reply cluster/images/etcd/Makefile:34]: Done

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Comment by serathius** [comment]
@pacoxu do you want to continue work on releasing Etcd? next step would be to publish the image like https://github.com/kubernetes/k8s.io/pull/2163

**Comment by pacoxu** [comment]
ok @serathius

send https://github.com/kubernetes/k8s.io/pull/2236

**Pull request merged**